### DISCUSSION TITLE
Icon V1-M/V1-X

### DISCUSSION THREAD

**Discussion by Hiddentalentpro** [bjoluc/cubase-mcu-midiremote#22] [Ideas]
I've had my Icon V1-M for a little less than a month now and was wondering if anybody out there has attempted to work on a MIDI remote patch to get all of the features working with Cubase. Mackie protocol works fine, just no channel colors, no master out meters and second display screen only shows CH01, CH02 CH03...

**Comment by bjoluc** [comment]
Hi @Hiddentalentpro, I'd really like to add support for the V1-M/V1-X but it's hard for me to do without having the hardware available. I might talk to Icon if they are willing to borrow me a unit for this. I'll get back to you when I know more :slightly_smiling_face:

**Reply by Hiddentalentpro** [reply]
That would be awesome! I don't know anything about writing code, but if I could be of help in any way, let me know!
